src/components/ConnectedFilter/utils.js

```diff
@@ -28,8 +28,14 @@ const getSingleFilterOption = (histogramResult, initHistogramRes) => {
     });
     return rangeOptions;
   }
-
-  const textOptions = histogramResult.histogram.map((item) => ({
+  let rawtextOptions = histogramResult.histogram;
+  // hide explorerHideDataFilter from filters
+  // explorerHideDataFilter added to guppyConfig in data-portal
+  if (explorerHideDataFilter) {
```

Comment:
made a comment in portal PR about making this variable an array, see https://github.com/uc-cdis/data-portal/pull/837#discussion_r616210052